src/generators/app/index.js

```diff
@@ -45,6 +42,23 @@ module.exports = generator.Base.extend({
       camelCaseName: _.camelCase(this.appname)
     };
 
-    this.fs.copyTpl([this.templatePath('**/*'), this.templatePath('**/.*')], '', vars);
-  }
+    var input = [this.templatePath('**/*'), this.templatePath('**/.*')];
+    return this.fs.copyTpl(input, '', vars);
+  },
+
+  initGitRepo: function () {
+    this.composeWith(
+      'git-init',
+      {
+        options: { commit: `Initialize Kibana Plugin (${pkg.name} v${pkg.version})` }
+      },
+      {
+        local: require.resolve('generator-git-init'),
+      }
+    );
+  },
```

Comment:
I don't see how this is the plugins responsibility. I find it strange that tools want to commit for me. Or maybe it's a usual thing with Yeoman? Never used it before